flight/targets/quanton/board-info/board_hw_defs.c

```diff
@@ -1646,7 +1646,284 @@ static const struct pios_tim_channel pios_tim_servoport_rcvrport_pins[] = {
 	},
 };
 
+/*
+ * 	OUTPUTS with extra outputs on receiverport
+    1:  TIM3_CH1  (PB4)
+    2:  TIM3_CH2  (PB5)
+    3:  TIM3_CH3  (PB0)
+    4:  TIM3_CH4  (PB1)
+    5:  TIM12_CH1 (PB14)
+    6:  TIM12_CH2 (PB15)
+    7:  TIM10_CH1 (PB8)
+    8:  TIM11_CH1 (PB9)
+    9:  TIM1_CH3  (PA10)    (IN1)
+    10: TIM8_CH1  (PC6)		(IN2)
+    11: TIM8_CH2  (PC7)		(IN3)
+    12: TIM8_CH3  (PC8)		(IN4)
+    13: TIM2_CH1  (PA15)	(IN5)
+    14: TIM2_CH2  (PB3)		(IN6)
+    15: TIM5_CH1  (PA0)		(IN7)
+    16: TIM5_CH2  (PA1)		(IN8)
+ */
 
+static const struct pios_tim_channel pios_tim_servoport_rcvrport_pins[] = {
```

Comment:
I've inspected the contents of this, and it looks good.

One comment though-- our standard is for C flight code to be indented per Linux kernel coding standards;  specifically, hte contents of this structure should be indented using tabs rather than spaces.

Once the structure is indented properly, I'll re-review and will be glad to merge.